lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -796,6 +789,14 @@ protected function getRenameIndexSQL($oldIndexName, Index $index, $tableName)
         ];
     }
 
+    private function quoteSingleIdentifierAsStringLiteral(string $levelName): string
+    {
+        $levelName = preg_replace('~^\[|\]$~s', '', $levelName);
```

Comment:
Why would one need to pass an explicitly quoted identifier when using SQL Server? Unlike Oracle, quoting doesn't affect the semantics of the identifier.